Automattic/woocommerce-payments
# Migrate existing tests to react testing library

Closes #585 

This PR is a base for the future work on migrating existing tests to RTL.
It allows migrating other tests in smaller chunks. 

Note: while this PR is in review all other migration PRs can be merged into this one.

#### Changes proposed in this Pull Request

* Add react testing library with jest configs
* Migrate evidence form tests from enzyme to RTL

#### Testing instructions

* Install dependencies with `npm install`.
* Run tests with `npm run test`. All tests should pass.

-------------------
